server/src/routes/v1/apps/handlers/getSingleApp.js

```diff
@@ -91,6 +91,8 @@ module.exports = {
             throw Boom.notFound('No apps found')
         }
 
-        return v1FormattedArray[0]
+        const result = v1FormattedArray[0]
+        result.userCanEditApp = isDeveloper
```

Comment:
this is the easiest way I found to be able to decide if a user should be able to edit, without making an extra call